Resources/Locale/en-US/prototypes/catalog/fills/crates/fun-crates.ftl

```diff
@@ -1,12 +1,24 @@
 ent-CrateFunPlushie = Plushie crate
     .desc = A buncha soft plushies. Throw them around and then wonder how you're gonna explain this purchase to NT.
 
-ent-CrateFunInstruments = Big band instrument collection
-    .desc = Get your sad station movin' and groovin' with this fine collection! Contains thirteen different instruments.
+ent-CrateFunInstrumentsVariety = Variety instrument collection
+    .desc = Get your sad station movin' and groovin' with this catch-all variety pack! Contains seven different instruments.
 
-ent-CrateFunBrass = Brass instrument ensemble crate
+ent-CrateFunInstrumentsBrass = Brass instrument ensemble crate
     .desc = Bring some jazz to the station with the brass ensemble. Contains a variety of brass instruments for the whole station to play.
 
+ent-CrateFunInstrumentsString = String instrument ensemble crate
+    .desc = Pluck or pick, slap or shred! Play a smooth melody or melt peoples' faces with this package of stringed instruments.
+
+ent-CrateFunInstrumentsWoodwind = Woodwind instrument ensemble crate
+    .desc = If atmos is good at their job, use air to play music with these woodwind instruments! Real wood not guaranteed with every item.
+
+ent-CrateFunInstrumentsKeyedPercussion = Keyed/Percussion instrument ensemble crate
+    .desc = Hit some keys with some sticks or your hands, with this Keyed and Percussion instrument ensemble crate.
+
+ent-CrateFunInstrumentsSpecial = Special instrument collector's crate
+    .desc = Create some noise with this special collection of arguably-instruments! Centcomm is not responsible for any trauma caused by the contents.
+
```

Comment:
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
who did this??
upstream isn't supposed to be using entity translation keys but oh well i guess?